weave/wandb_interface/wandb_lite_run.py

```diff
@@ -36,47 +63,53 @@ class InMemoryLazyLiteRun:
 
     # Property Cache
     _i_api: typing.Optional[InternalApi] = None
-    _run: typing.Optional[Run] = None
+    _run: typing.Optional[public.Run] = None
     _stream: typing.Optional[file_stream.FileStreamApi] = None
     _pusher: typing.Optional[FilePusher] = None
-    _use_async_file_stream: bool = False
 
     def __init__(
         self,
         entity_name: str,
         project_name: str,
         run_name: typing.Optional[str] = None,
+        config: typing.Optional[typing.Dict] = None,
         *,
         job_type: typing.Optional[str] = None,
         group: typing.Optional[str] = None,
         _hide_in_wb: bool = False,
-        _use_async_file_stream: bool = False,
     ):
-        wandb_client_api.assert_wandb_authenticated()
+        assert_wandb_authenticated()
 
         # Technically, we could use the default entity and project here, but
         # heeding Shawn's advice, we should be explicit about what we're doing.
         # We can always move to the default later, but we can't go back.
         if entity_name is None or entity_name == "":
-            raise ValueError(f"Must specify entity_name")
+            raise ValueError("Must specify entity_name")
         elif project_name is None or project_name == "":
-            raise ValueError(f"Must specify project_name")
+            raise ValueError("Must specify project_name")
 
         self._entity_name = entity_name
         self._project_name = project_name
         self._display_name = run_name
         self._run_name = run_name or runid.generate_id()
+        self._config = config or dict()
+        # TODO: wire up actual telemetry?
+        self._config.update(
+            {
+                "_wandb": {
+                    "streamtable_version": "0.1",
+                    "cli_version": __version__,
+                    "python_version": platform.python_version(),
+                    "is_jupyter_run": _get_python_type() != "python",
+                },
+            }
+        )
```

Comment:
This throws some basic telemetry into `run.config._wandb`.  We should wire up actual wandb sdk telemetry here but at least we have something.